app/assets/javascripts/admin/addon/templates/user-fields.hbs

```diff
@@ -1,3 +1,4 @@
+{{! template-lint-disable no-array-prototype-extensions }}
```

Comment:
Similarly... did we introduce some new rules? This wasn't causing any violation before 🤔